ui/src/app/pages/workspace/workspace-wrapper/component.ts

```diff
@@ -67,6 +73,7 @@ export class WorkspaceWrapperComponent implements OnInit, OnDestroy {
           this.setHelpContent();
           if (this.helpContent !== 'reviewParticipantDetail') {
             this.sidebarOpen = false;
+            localStorage.setItem('sidebarState', 'closed');
```

Comment:
also here.

nit - IMO, it's easier to maintain the syncing to localStorage if its done in a one place (the setter) and the rest of the file just uses the setter and doesn't have to worry about localStorage.

Reply:
Agreed, much cleaner this way.